css/index.css

```diff
@@ -0,0 +1,311 @@
+  position: relative;
+  top: 100px;
+  left: 300px;
+  font-size: 300%;
+}
+
+#hr_one {
+  width: 700px;
+  position: relative;
+  top: 80px;
+  left: 170px;
+}
+
+.article_contact {
+  display: flex;
+  justify-content: flex-start;
+  list-style: none;
+  margin-top: 100px;
+  position: relative;
+  left: 280px;
+}
+
+.contact_link {
+  display: inline-block;
+  margin: 8px;
+  border: solid 1px #eee;
+  background-color: #eee;
+  border-radius: 50%;
+}
+
+.contact_link:hover {
+  border-color: black;
+}
+
+.articl_img {
+  display: block;
+  width: 945px;
+  height: 531px;
+  position: relative;
+  left: 170px;
+}
+
+.img_desc {
+  display: block;
+  width: 700px;
+  height: 35px;
+  position: relative;
+  left: 170px;
+  font-family: georgia, 'times new roman', times, serif;
+  color: #666;
+}
+
+.img_comment {
+  display: block;
+  width: 150px;
+  position: relative;
+  top: 80px;
+  left: 300px;
+  color: #333;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+#img_comment_link {
+  text-decoration: none;
+  color: #333;
+}
+
+#img_comment_link:hover {
+  text-decoration: underline;
+}
+
+#hr_two {
+  width: 700px;
+  position: relative;
+  top: 80px;
+  left: 170px;
+}
+
+.article_sign_but {
+  position: relative;
+  top: 90px;
+  left: 900px;
+}
+
+.article_sign {
+  width: 494px;
+  height: 41px;
+  position: relative;
+  left: 300px;
+  top: 50px;
+}
+
+.para_one {
+  display: flex;
+  top: 50px;
+  position: relative;
+  left: 300px;
+  width: 600px;
+  height: 120px;
+}
+
+.para {
+  display: flex;
+  position: relative;
+  left: 300px;
+  width: 600px;
+  height: 120px;
+  top: -20px;
+}
+
+.hr_three {
+  display: flex;
+  position: relative;
+  top: -20px;
+  width: 700px;
+  left: 170px;
+}
+
+.hr_four {
+  display: flex;
+  position: relative;
+  top: -120px;
+  width: 700px;
+  left: 170px;
+}
+
+#top_para {
+  margin-top: -100px;
+}
+
+.para_seven {
+  width: 600px;
+  position: relative;
+  left: 300px;
+  font-family: georgia, 'times new roman', times, serif;
+}
+
+.rm_txt_decor {
+  text-decoration: none;
+  color: #333;
+}
+
+.rm_txt_decor:hover {
+  text-decoration: underline;
+}
+
+.para_ten_title {
+  display: flex;
+  position: relative;
+  left: 300px;
+}
+
+.second_article {
+  display: block;
+  position: relative;
+  width: 950px;
+  height: 600px;
+  right: 800px;
+  top: 2600px;
+}
+
+.first_element {
+  display: block;
+  width: 210px;
+}
+
+.element_one_img {
+  display: flex;
+  width: 210px;
+}
+
+.second_element {
+  display: block;
+  width: 210px;
+  position: relative;
+  left: 220px;
+  top: -327px;
+}
+
+.element_two_img {
+  display: flex;
+  width: 210px;
+}
+
+.third_element {
+  display: block;
+  width: 210px;
+  position: relative;
+  left: 450px;
+  top: -626px;
+}
+
+.element_third_img {
+  display: flex;
+  width: 210px;
+}
+
+.fourth_element {
+  display: block;
+  width: 210px;
+  position: relative;
+  left: 700px;
+  top: -911px;
+}
+
+.element_fourth_img {
+  display: flex;
```

Comment:
[Required] CSS should be DRY (don't repeat yourself) => 

=> This property `display:flex;` is repeated 20 times, this is a bad practice, kindly use utility classes or group together elements which have the same property

=> Look for all of repeated properties and fix them like that